mrmustard/lab_dev/circuits.py

```diff
@@ -427,6 +430,45 @@ def validate_path(self, path) -> None:
 
             remaining[i1] = (remaining[i1] @ remaining.pop(i2))[0]
 
+    def serialize(self, filestem=None):
```

Comment:
```suggestion
    def serialize(self, filestem: str=None):
```
Maybe even optionally a Path?